pkg/issuer/acme/http/http.go

```diff
@@ -328,6 +328,10 @@ func (s *Solver) ensurePod(crt *v1alpha1.Certificate, domain, token, key string,
 							corev1.ResourceCPU:    resource.MustParse("10m"),
 							corev1.ResourceMemory: resource.MustParse("2Mi"),
 						},
+						Limits: corev1.ResourceList{
+							corev1.ResourceCPU:    resource.MustParse("10m"),
+							corev1.ResourceMemory: resource.MustParse("2Mi"),
```

Comment:
These limits are too low I think. The pod is OOM'ing because it is hitting the 2MB memory limit. Maybe increase this to 64Mi? No harm it giving plenty of room just in case it needs it.